client/cypress/integration/authentication/sign-in.spec.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-named-as-default,import/no-named-as-default-member
+import SignInPage from '../../model/SignInPage';
+import { validUser } from '../../fixtures/users';
+// eslint-disable-next-line import/no-named-as-default,import/no-named-as-default-member
```

Comment:
YOU SHOULD NOT DISABLE RULES IF THEY ARE SETUP IN YOUR REPOSITORY!
3 ways for not to get Eslint barking:
- follow rule (highly recommended);
- disable rule in Eslint config; 
- setup plugin which allows to use desired syntax (i'm not sure it is possible).